### PR TITLE
Update 22_Process-Team-Organization.md

### LINKED ISSUE #210: 2.2 - Process Team Organization

### PR DESCRIPTION
**Pull request by Jakobis**
Not sure if I think this is good enough, but I have a hard time thinking of pros and cons of our team organization.

### CODE REVIEW

**Review by tobloef — DISMISSED** [review]
Kunne du prøve at omformulere/omstrukturere den sidste sætnind?

Brug desuden ikke vi/jeg i en rapport.

**Comment by Jakobis** [comment]
@tobloef Har rettet så vidt muligt efter dine kommentarer

File: report/22_Process-Team-Organization.md [hunk 1]
```diff
@@ -1 +1,5 @@
-How is the team organized?
+The team has organized itself mainly through a weekly meeting after the lecture in which tasks were distributed. The team has not found any formalized process corresponding to this, but it has had several advantages and drawbacks.
```
Comment (Hjaltesorgenfrei) [inline report/22_Process-Team-Organization.md]: This starts in the same way as 2.1 I would reword it here.
I would not start the next sentence with "The Team", you cant just swap we out with the team. It has the same awkwardness. 

> No formal process corresponding to our method of work could be found.

Also the method needs more explanation than a single line saying we only organized through that meeting as that simply is not true. For example we had no leader and every member in the team was equal. We worked when we had time. 
Assigning work using github issues is also a method of Team Organization as its clear who works on what. We also avoided working on each other branches and let changes needed to be done to a PR be done by the developer who made the PR.

File: report/22_Process-Team-Organization.md [hunk 2]
```diff
@@ -1 +1,5 @@
...
+
+On the positive side, this method has very little overhead, and requires little scheduling since the main activity is done in continuation of a mandatory activity for which members are supposed to be present. It also leaves more time for actual development work.
```
Comment (Hjaltesorgenfrei) [inline report/22_Process-Team-Organization.md]: Does not really mean anything as the method was not explained in full.

File: report/22_Process-Team-Organization.md [hunk 3]
```diff
@@ -1 +1,5 @@
-How is the team organized?
+The team has organized itself mainly through a weekly meeting after the lecture in which tasks were distributed. The team has not found any formalized process corresponding to this, but it has had several advantages and drawbacks.
+
+On the positive side, this method has very little overhead, and requires little scheduling since the main activity is done in continuation of a mandatory activity for which members are supposed to be present. It also leaves more time for actual development work. 
+
+The method does, however, suffer a few problems. It does not involve any dedicated time for working, which makes it harder to schedule activities like pair programming, which has been done a few times throughout the course. There is no mechanic to ensure that more overreaching goals are reached, which by coincidence was not a major problem in this course. This is likely due to the fact that the team managed to get through quite a few tasks in the first few weeks, and could thus keep up with the scheduled tasks.
```
Comment (Hjaltesorgenfrei) [inline report/22_Process-Team-Organization.md]: When did we pair program?
I would argue that there was a mechanic to ensure that the "overreaching goals" (assigned tasks from teachers right) were reached. We were given them at the end of the lecture and made a plan for how to get them done by next week. Also people were haunted in the Discord and until they did their assigned tasks.

**Review by Hjaltesorgenfrei — CHANGES_REQUESTED** [review]
Really needs to be fleshed out

File: report/22_Process-Team-Organization.md (hunk 3 above)
Comment (Jakobis) [reply report/22_Process-Team-Organization.md]: I have paired with Tobias at least two times. I don't know if anybody else have done it.

**Review by Hjaltesorgenfrei — APPROVED** [review]
